src/StatPlots.jl

```diff
@@ -7,6 +7,9 @@ import Plots: _cycle
 using StatsBase
 using Distributions
 using DataFrames
+using IterableTables
+import DataValues: DataValue
```

Comment:
I think you could also implement these from IterableTables - wouldn't that be cleaner?

Reply:
good point

Reply:
I gotta say your macro-fu is pretty on point :-)

Reply:
Actually, I'd leave it like it is, see my other comments.